lib/auth/auth.go

```diff
@@ -105,7 +105,7 @@ func NewServer(cfg *InitConfig, opts ...ServerOption) (*Server, error) {
 		cfg.Trust = local.NewCAService(cfg.Backend)
 	}
 	if cfg.Presence == nil {
-		cfg.Presence = local.NewPresenceService(cfg.Backend)
+		cfg.Presence = local.NewPresenceServiceV2(cfg.Backend, cfg.AppPublisher)
```

Comment:
do you want to name it IdemeumNewPresenceService?

Reply:
Will rename it, thanks